app/controllers/loans_controller.rb

```diff
@@ -1,20 +1,22 @@
 class LoansController < ApplicationController
-  skip_before_action :authenticate_request, only: [ :create ]
+  skip_before_action :authenticate_request, only: [ :create, :show, :index ]
   before_action :set_loan, only: [ :show, :update, :destroy ]
 
   def index
     @loans = Loan.all
-    render json: @loans, status: :ok
+    render json: @loans, each_serializer: LoanSerializer, status: :ok
   end
 
   def show
-    render json: @loan, status: :ok
+    render json: @loan, serializer: LoanSerializer, status: :ok
   end
 
   def create
     @loan = Loan.create(set_params)
+    LoanSerializer.new(@loan)
```

Comment:
unnecessary initialisation of loanserializer. Line no 20, It will automatically call LoanSerializer.new(@loan). so remove this line.